unitTests/ui/ResultListTest.ts

```diff
@@ -784,11 +784,48 @@ export function ResultListTest() {
         describe('enableScrollToTop set to', () => {
           let infiniteScrollContainer: HTMLElement;
           let allResult: IQueryResults;
+          let scrollToSpy: jasmine.Spy;
+          let scrollTopSpy: jasmine.Spy;
+
           beforeEach(() => {
-            infiniteScrollContainer = { scrollTop: 500 } as HTMLElement;
+            document.createElement('div');
+            infiniteScrollContainer = document.createElement('div');
+            infiniteScrollContainer.style.height = '10000px';
+            infiniteScrollContainer.style.overflow = 'scroll';
+            document.body.appendChild(infiniteScrollContainer);
```

Comment:
Would this test still be doable without appending to the document ?

Could it be added to a fake container, without touching the document ?

Reply:
Artifact of an old version of the tests will remove it.